cmd/testrun_abort.go

```diff
@@ -21,10 +21,35 @@ var (
 			}
 		},
 	}
+	testRunAbortAllOpts struct {
+		Organisation string
+	}
+
+	testRunAbortAllCmd = &cobra.Command{
+		Use:   "abort-all <organisation-ref>",
+		Short: "Abort all running tests for a given organisation",
+		Long:  "Abort all running tests for a given organisation",
+		Run:   testRunAbortAll,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if len(args) > 1 {
+				log.Fatal("Too many arguments")
+			}
+			if len(args) < 1 {
+				log.Fatal("Missing organisation")
+			}
```

Comment:
You can make your life easier by settings the `Args` property ont he `cobra.Command` to `cobra.ExactArgs(1)`.

See https://pkg.go.dev/github.com/spf13/cobra#readme-positional-and-custom-arguments